pkg/pushsync/pushsync.go

```diff
@@ -35,32 +35,35 @@ type Receipt struct {
 }
 
 type PushSync struct {
-	streamer      p2p.Streamer
-	storer        storage.Putter
-	peerSuggester topology.ClosestPeerer
-	tagg          *tags.Tags
-	logger        logging.Logger
-	metrics       metrics
+	streamer         p2p.Streamer
+	storer           storage.Putter
+	peerSuggester    topology.ClosestPeerer
+	tagg             *tags.Tags
+	deliveryCallback func(swarm.Chunk) error// callback func to be invoked to deliver chunks to PSS
```

Comment:
gofmt would add a whitespace char before this comment

Reply:
i am not sure which version are you looking.. i have a space here...

Reply:
i was looking at the version which was up at the time of submitting my review. this is why the `outdated` tag is present.